### PR TITLE
feature: balance testing cmd

### PR DESCRIPTION
**Pull request by ycdesu**
Sometimes the document isn't that clear, so it will be better to have a command to test the function. I add a cmd to print your balance of this account. In the future I will add websocket related commands.

### CODE REVIEW

File: cmd/ftx/main.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/c9s/bbgo/pkg/cmd/ftx"
+)
+
+// We could use this command to test FTX functions.
+func main() {
```
Comment (c9s) [inline cmd/ftx/main.go:10]: how about:

    bbgo balance --session ftx

Comment (ycdesu) [reply cmd/ftx/main.go]: Good idea. This PR is ready for another review, thanks

**Comment by ycdesu** [comment]
Now we could print the balance by `godotenv -f .env.local go run ./cmd/bbgo balances --session=ftx`. Currently only support ftx.

**Comment by c9s** [comment]
> Now we could print the balance by `godotenv -f .env.local go run ./cmd/bbgo balances --session=ftx`. Currently only support ftx.

actually we have embedded the godotenv in the root command, you can check that.

abd we also provide a builtin persistent option named `--dotenv`

**Comment by c9s** [comment]
by default, bbgo looks for the ".env.local" file, so you can ignore the dotenv command wrapper now.

**Pull request merged**